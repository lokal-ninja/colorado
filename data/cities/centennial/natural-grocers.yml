id: 787346902
type: way
shop: greengrocer
latitude: 39.5957437
longitude: -104.85271970000001
postcode: "80112"
city: Centennial
street: "East Arapahoe Road"
housenumber: 11753
phone: ""
opening_hours: ""
website: ""
