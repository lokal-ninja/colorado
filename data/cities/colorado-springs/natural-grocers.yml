id: 941877524
type: node
latitude: 38.81144
longitude: -104.82344
postcode: "80906"
city: Colorado Springs
street: South Nevada Avenue
housenumber: "1604"
phone: +1-719-634-9200
opening_hours: Mo-Sa 07:57-21:06; Su 08:57-19:36
website: https://www.naturalgrocers.com/store/colorado-springs-south-nevada-ave
