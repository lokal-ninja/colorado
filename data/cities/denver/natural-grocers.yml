id: 3374384041
type: node
latitude: 39.7698814
longitude: -105.044584
postcode: "80212"
city: Denver
street: "Tennyson Street"
housenumber: 3825
phone: ""
opening_hours: ""
website: ""
