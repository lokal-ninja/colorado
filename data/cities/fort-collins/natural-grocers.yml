id: 2702160653
type: node
latitude: 40.5282885
longitude: -105.0751963
postcode: "80525-3021"
city: Fort Collins
street: "South College Avenue"
housenumber: 4318
phone: ""
opening_hours: "Mo-Sa 08:56-20:04; Su 08:56-19:06"
website: ""
