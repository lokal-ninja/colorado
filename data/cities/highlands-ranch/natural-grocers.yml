id: 1758055558
type: node
latitude: 39.55042
longitude: -105.00311
postcode: "80129"
city: Highlands Ranch
street: "Sergeant Jon Stiles Drive"
housenumber: 1265
phone: "+1 (303) 471-9400"
opening_hours: "Mo-Sa 08:30-21:05, Su 09:00-20:05"
website: "https://www.naturalgrocers.com/store/highlands-ranch"
