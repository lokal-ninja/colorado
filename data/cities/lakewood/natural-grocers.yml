id: 724695780
type: node
latitude: 39.70234
longitude: -105.1382
postcode: "80228"
city: Lakewood
street: West Alameda Parkway
housenumber: "12612"
phone: +1-303-986-5700
opening_hours: Mo-Sa 08:27-20:36; Su 08:57-19:36
website: https://www.naturalgrocers.com/store/lakewood-green-mountain
