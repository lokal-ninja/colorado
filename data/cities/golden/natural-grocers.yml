id: 92737034
type: way
shop: supermarket
latitude: 39.747095900000005
longitude: -105.2091439
postcode: ""
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""
